hardhat.config.js

```diff
@@ -1,16 +1,16 @@
-require("@nomiclabs/hardhat-waffle");
+require("@nomicfoundation/hardhat-toolbox");
 
 // The next line is part of the sample project, you don't need it in your
 // project. It imports a Hardhat task definition, that can be used for
 // testing the frontend.
 require("./tasks/faucet");
 
-// If you are using MetaMask, be sure to change the chainId to 1337
+/** @type import('hardhat/config').HardhatUserConfig */
 module.exports = {
   solidity: "0.7.3",
   networks: {
     hardhat: {
-      chainId: 31337
+      chainId: 31337 // If you are using MetaMask, be sure to change the chainId to 1337
```

Comment:
```suggestion
      chainId: 1337 // We set 1337 to make interacting with MetaMask, simpler
```

NOTE: Make sure the tutorial also works well with this.

Reply:
I confirmed that this does work, _if_ your change [here](https://github.com/NomicFoundation/hardhat-boilerplate/pull/63/files#diff-1d68a434848d82c8522e743ac6deaafa553eac04b2f6c84645cdf398b17c6473R25) is also included.